app/src/seed_script.py

```diff
@@ -0,0 +1,195 @@
+import os
+from .models import db, Course, Module, Topic, Lesson, Subject, User, Points, UserProgress, GradeEnum
+
```

Comment:
Presuming you've already created the placeholder folders and moved the files within them (see main comment), add these for setting up changes in `create_file_paths()` below:

```python
from shutil import copyfile

VIDEO_PLACEHOLDER = "static/vendor/lesson-videos/placeholder/video.mp4"
THUMBNAIL_PLACEHOLDER = "static/vendor/lesson-videos/placeholder/thumbnail.png"
TEXTBOOK_PLACEHOLDER = "static/vendor/textbooks/placeholder/sample.pdf"

```